pyhive/tests/test_sqlalchemy_hive.py

```diff
@@ -112,15 +134,15 @@ def test_reflect_select(self, engine, connection):
     @with_engine_connection
     def test_type_map(self, engine, connection):
         """sqlalchemy should use the dbapi_type_map to infer types from raw queries"""
-        row = connection.execute('SELECT * FROM one_row_complex').fetchone()
+        row = connection.execute(text('SELECT * FROM one_row_complex')).fetchone()
         self.assertListEqual(list(row), _ONE_ROW_COMPLEX_CONTENTS)
 
     @with_engine_connection
     def test_reserved_words(self, engine, connection):
         """Hive uses backticks"""
         # Use keywords for the table/column name
-        fake_table = Table('select', MetaData(bind=engine), Column('map', sqlalchemy.types.String))
-        query = str(fake_table.select(fake_table.c.map == 'a'))
+        fake_table = Table('select', MetaData(), Column('map', sqlalchemy.types.String))
+        query = str(fake_table.select().where(fake_table.c.map == 'a').compile(engine))
```

Comment:
To get the statement as compiled to a specific dialect or engine, you have to use compile() method.

https://docs.sqlalchemy.org/en/20/core/foundation.html#sqlalchemy.sql.expression.ClauseElement.compile